docker: openwrt: pass build-arg variables explicitly

arnout opened the pull request:
When the --build-arg option is given without a value for the argument,
docker will use an environment variable with the same name. However,
podman (a "drop-in" replacement for docker with better non-privileged
support) doesn't behave like that. Therefore, give the arguments
explicitly.

rmelotte commented on tools/docker/builder/openwrt/build.sh:
```diff
@@ -174,6 +174,7 @@ main() {
 
 }
 
+VERBOSE=true
```
This change is not mentioned in the commit message.
I just wanted to mention there was a --verbose option before that set this variable, it has been removed with the shellcheck fixes.
I added the option back in my upcoming PR, but we could also not add the option back and keep this change (as you prefer).

arnout replied:
That was actually a debugging leftover that I forgot to remove again...

arnout committed:
docker: openwrt: pass build-arg variables explicitly

When the --build-arg option is given without a value for the argument,
docker will use an environment variable with the same name. However,
podman (a "drop-in" replacement for docker with better non-privileged
support) doesn't behave like that. Therefore, give the arguments
explicitly.

Signed-off-by: Arnout Vandecappelle (Essensium/Mind) <arnout@mind.be>
